contracts/StablecoinConverter.sol

```diff
@@ -318,6 +318,14 @@ contract StablecoinConverter is EpochTokenLocker {
         return IdToAddressBiMap.getAddressAt(registeredTokens, id);
     }
 
+    /** @dev View returning a bool attesting whether token was already added
+      * @param addr address of the token to be checked
+      * @return bool attesting whether token was already added
+      */
+    function hasToken(address addr) public view returns (bool) {
```

Comment:
Could have reused `tokenAddressToIdMap` and check for `≠ MAX_UINT`?

Reply:
Yes, definitively. But I guess this new interface is much easier to use for someone not so familiar with the code